airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -54,6 +88,31 @@ const ConnectorList: React.FC<MenuWithRequestButtonProps> = ({
   </>
 );
 
+const Option: React.FC<OptionProps> = (props) => {
+  return (
+    <components.Option {...props}>
+      <OptionView
+        data-testid={props.data.label}
+        isSelected={props.isSelected}
+        isDisabled={props.isDisabled}
+      >
+        <Text>
+          {props.data.img || null}
+          <Label>{props.label}</Label>
+        </Text>
+        {props.data.releaseStage ? (
+          <Stage isNew={props.data.releaseStage === releaseStage.NEW_RELEASE}>
```

Comment:
`New` should be ignored for this release (see https://github.com/airbytehq/airbyte/issues/9711#issuecomment-1024722413), since `NEW_RELEASE` also doesn't exist on the return value.